chipsec_main.py

```diff
@@ -399,7 +399,7 @@ def parse_args(self):
             logger().DEBUG   = True
         if self.log:
             logger().set_log_file( self.log )
-        if self._module_argv and len(self._module_argv) == 1 and len(self._module_argv[0].split(',')) > 1:
+        if self._module_argv and len(self._module_argv) == 1 and len(self._module_argv[0].count(',')) >= 1:
```

Comment:
`count()` already returns the number of commas, no need to call `len()` on it. And since you only need to check for _one_ comma, something like `',' in self._module_argv[0]` is even better IMO.

Reply:
Agree, I missed the len call still being in the commit.